### PR TITLE
[*] Smart Actions - Add a missing step in Smart Action download example configuration

### PR DESCRIPTION
**Pull request by arnaudbesnier**
Also fix: https://github.com/ForestAdmin/forest-doc/issues/41

### CODE REVIEW

File: _includes/rails/actions.md [hunk 1]
```diff
@@ -232,6 +232,38 @@ end
...
+        methods: :any,
+        headers: :any,
+        expose: ['Content-Disposition']
+    end
+  end
+end
+```
+
+<div class="c-notice warning l-mt">
+  ⚠️ If you forget to specify "expose: ['Content-Disposition']" in the CORS configuration, the filename of the downloaded file won't be customisable as described in the step 4.
```
Comment (Tolsa) [inline _includes/rails/actions.md]: "following step 4" plutôt et pourquoi pas mettre un petit lien vers l'ancre du 4. Comme on fournit pas mal d'infos en plus y'a moyen que quand l'utilisateur lise le step 2, il ne voit pas encore le 4 à l'écran. C'est pas hyper bloquant mais plus on le prend par la main mieux c'est.

File: _includes/rails/actions.md [hunk 2]
```diff
@@ -258,7 +290,8 @@ end
 ```ruby
 class Forest::CustomersController < ForestLiana::ApplicationController
   def download_file
-    data = open('https://.../file.pdf')
+    # TODO: Add a "myLocalFile.pdf" file at the root of your project to have a working example.
```
Comment (Tolsa) [inline _includes/rails/actions.md]: Comme ma dernière review plus haut, à voir si c'est pertinent de rementionner l'alerte importante du point 2, car c'est complètement lié à la manip CORS.

Comment (arnaudbesnier) [reply _includes/rails/actions.md]: As the documentation is open-source, can we use english to be sure that a majority of people visiting this PR will understand the discussion. 

The change, here, is a fix of the current example about to open a file before sending it.

File: _includes/rails/actions.md (hunk 1 above)
Comment (arnaudbesnier) [reply _includes/rails/actions.md]: In my opinion, it is not a good idea to add a link here, as a beginner, if you click on it you could be lost in the example steps. The possibility to bypass steps can generate implementation errors from developers that the customer support would have to deal with. 

But If you really want it, I'll let you adapt the code to make you feel comfortable with it.

Comment (Tolsa) [reply _includes/rails/actions.md]: Ok, your argument in this case of bypassing step 3 due to a click on this warning message makes a lot of sense. Only my wording remark stays relevant, but it is merely a detail :)

**Pull request merged**